main.go

```diff
@@ -208,32 +227,53 @@ job "dir2consul-{{ .GitRepoName }}" {
         task "dir2consul" {
             artifact {
                 destination = "local/{{ .GitRepoName }}"
-				source = "{{ .GitRepoURL }}"
+                source = "git::{{ .GitRepoURL }}"
                 options {
                     sshkey = "{{ .SshKey }}"
                 }
             }
             config {
-                image = "jimrazmus/dir2consul:v1.4.1"
+                image = "jimrazmus/dir2consul:rc"
```

Comment:
Use current release.
```suggestion
                image = "code42software/dir2consul:v1.5.0"
```